package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
You should have this package.json inside `02-oop-inheritance` in order to have one for each topic.